src/Resources/config/listener.yml

```diff
@@ -41,6 +41,12 @@ services:
             # and lower than the header replay header (defaults to 7)
             - { name: kernel.event_listener, event: kernel.request, method: onKernelRequest, priority: 6 }
 
+    contao.listener.clear_form_data:
+        class: Contao\CoreBundle\EventListener\ClearFormDataListener
+        tags:
+            # The priority must be lower than the one of the Symfony save session listener (defaults to -1000)
```

Comment:
the priority must be **higher** than...

Reply:
Fixed in 6847201f50a61d2fc69a8907d67074a7377d4396.